forge/src/main/java/hardcorequesting/forge/HardcoreQuestingForge.java

```diff
@@ -218,23 +205,26 @@ public void registerOnClientTick(Consumer<Minecraft> consumer) {
     
     @Override
     public void registerOnWorldTick(Consumer<Level> consumer) {
-        MinecraftForge.EVENT_BUS.<TickEvent.WorldTickEvent>addListener(event -> {
+        MinecraftForge.EVENT_BUS.<TickEvent.LevelTickEvent>addListener(event -> {
             if(event.phase == TickEvent.Phase.END)
-                consumer.accept(event.world);
+                consumer.accept(event.level);
         });
     }
-    
+
+    //Todo come back to this
```

Comment:
`registerOnHudRender()` appears to be unused in recent versions of HQM. Perhaps you could resolve this todo by removing the function?